soen341-UB3-QA/resources/views/welcome.blade.php

```diff
@@ -151,7 +151,7 @@
             </div>
             <br />
             <div>
-                <a href="/details" class="btn btn-primary">Question Page link</a>
+                <a href="/question/1" class="btn btn-primary">Question Page link</a>
             </div>
```

Comment:
will need to change "/question/1" to the appropriate id after when questions are linked to this page